JS/Lesson32/onur/my-app/src/components/RecipeList/index.jsx

```diff
@@ -1,12 +1,31 @@
-export const RecipeList = ({recipes}) => {
+import { useState } from "react";
+import { RecipeDetail } from "../RecipeDetail";
+import './styles.css'
+
+
+export const RecipeList = ({ recipes }) => {
+
+	const [recipe, setRecipe] = useState(false);
```

Comment:
I think this state is not needed at all. You should be able to use selectedRecipe for everything that you used `recipe` state for (see comments below).